app-modules/api/src/Http/Requests/EventApiV0Request.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace HackGreenville\Api\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class EventApiV0Request extends FormRequest
+{
+    public function authorize()
+    {
+        return true;
+    }
+
+    public function rules()
+    {
+        return [
+            'start_date' => [
+                'nullable',
+                'sometimes',
+                'date',
+                'before_or_equal::end_date',
```

Comment:
```suggestion
                'before_or_equal:end_date',
```
I'm still getting redirected with this colon out of the picture, but after looking [here](https://laravel.com/docs/10.x/validation#rule-before-or-equal) it looks like it should be removed.